python/tvm/auto_scheduler/search_task.py

```diff
@@ -118,14 +130,21 @@ def __init__(
         num_measure_trials=0,
         early_stopping=None,
         num_measures_per_round=64,
+        working_dir=None,
+        check_correctness=False,
         verbose=1,
         builder="local",
+        builder_n_parallel=-1,
```

Comment:
The `-1` is conflict with `None` default value before? Could we unify the default value (-1 or None) when users don't set.? Currently, I am a bit confused with two default values